### DISCUSSION TITLE
up-poll with up-params

### DISCUSSION THREAD

**Discussion by nicolagianelli** [unpoly/unpoly#645] [Q&A] [answered]
Hello, I tried to use the up-params attribute alongside the up-poll attribute but It didn't work.
I ended up adding the params manually to the up-source url but it would be better to have the up-params working with the up-poll also if possible.

Thank you for this library!

**Comment by triskweline** [comment, marked as answer]
This should be in the next release.

**Comment by triskweline** [comment]
This issue has been resolved with version 3.9.0.
